vsintegration/src/FSharp.Editor/CodeFix/ImplementInterfaceCodeFixProvider.fs

```diff
@@ -118,7 +118,7 @@ type internal FSharpImplementInterfaceCodeFixProvider
                             title,
                             (fun (cancellationToken: CancellationToken) ->
                                 async {
-                                    let! sourceText = context.Document.GetTextAsync() |> Async.AwaitTask
+                                    let! sourceText = context.Document.GetTextAsync(cancellationToken) |> Async.AwaitTask
```

Comment:
We should shadow, though in reality it's not particularly important to do that unless there is an intervening Async.Parallel, Async.StartChild or the like (and even then it's not that important TBH)